app/views/galleries/show.rb

```diff
@@ -52,7 +60,10 @@ def gallery_item_content(image)
         creator: presenter.field_value('dataProvider'),
         thumb_url: gallery_image_thumbnail(image),
         full_url: presenter.field_value('aggregations.edmIsShownBy'),
-        rights: presenter.simple_rights_label_data
+        rights: presenter.simple_rights_label_data,
+        url_item: image.portal_url,
+        url_collection: search_path(q: "europeana_collectionName:#{presenter.field_value('europeanaCollectionName')}"),
+        institution_logo: data_provider_logo_url(image)
```

Comment:
just pass the presenter, and you won't need to look up what the presenter is in data_provider_logo_url